docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -571,14 +571,14 @@ For more details on inventory plugins and dynamic inventory scripts see :ref:`in
 Connecting to hosts: behavioral inventory parameters
 ====================================================
 
-As described above, setting the following variables control how Ansible interacts with remote hosts.
+As described above, setting the following variables controls how Ansible interacts with remote hosts.
 
 Host connection:
 
 .. include:: shared_snippets/SSH_password_prompt.txt
 
 ansible_connection
-    Connection type to the host. This can be the name of any of ansible's connection plugins. SSH protocol types are ``smart``, ``ssh`` or ``paramiko``.  The default is smart. Non-SSH based types are described in the next section.
+    Connection type to the host. This can be the name of any of Ansible's connection plugins. SSH protocol types are ``smart``, ``ssh`` or ``paramiko``.  The default is smart. Non-SSH based types are described in the next section.
```

Comment:
```suggestion
    Connection type to the host. This can be the name of any Ansible connection plugin. SSH protocol types are ``smart``, ``ssh`` or ``paramiko``.  The default is smart. Non-SSH based types are described in the next section.
```